### PR TITLE
Update/field selector illustration

### PR DESCRIPTION
**Pull request by Reihaneh-Shahali**

### CODE REVIEW

File: src/components/Feedback/SelectField/SelectorCard/SelectorCard.tsx [hunk 1]
```diff
@@ -80,7 +81,7 @@ const ResultMenu = styled.div<MenuProps>`
 `;
 
 const ResultList = styled.div`
-  height: calc(
+  max-height: calc(
```
Comment (mariush2) [inline src/components/Feedback/SelectField/SelectorCard/SelectorCard.tsx:84]: Doesnt this cause tha MissingAccesses element to go to the top when searching?

Comment (Reihaneh-Shahali) [reply src/components/Feedback/SelectField/SelectorCard/SelectorCard.tsx:84]: yes

Comment (mariush2) [reply src/components/Feedback/SelectField/SelectorCard/SelectorCard.tsx:84]: Does it work with having max-height and height set?

Comment (Reihaneh-Shahali) [reply src/components/Feedback/SelectField/SelectorCard/SelectorCard.tsx:84]: yes I am adding both now. I made a mistake. you were right. Missing access is not at the bottom now.

**Review by mariush2 — APPROVED** [review]

**Pull request merged**